S/sed/build_tarballs.jl

```diff
@@ -12,15 +12,23 @@ sources = [
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/sed-4.8/
+cd $WORKSPACE/srcdir/sed-*
+if [[ "${target}" == *-mingw* ]]; then
+    # Fix error
+    #    sed/sed-compile.o: In function `sprintf':
+    #    /opt/x86_64-w64-mingw32/x86_64-w64-mingw32/sys-root/include/stdio.h:366: undefined reference to `__chk_fail'
+    # See https://github.com/msys2/MINGW-packages/issues/5868#issuecomment-544107564
+    export LDFLAGS="-lssp"
```

Comment:
```suggestion
    export LIBS="-lssp"
```
`LDFLAGS`: Don't use this variable to pass library names (-l) to the linker; use LIBS instead.